x-pack/plugins/cases/public/components/recent_cases/filters/index.tsx

```diff
@@ -49,12 +69,14 @@ export const RecentCasesFilters = React.memo<{
   );
 
   return (
-    <EuiButtonGroup
+    <EuiSuperSelect
```

Comment:
yeah sure, I don't mind using `EuiSelect`. Just curious, any specific reason to not use `EuiSuperSelect`?

Reply:
Nothing wrong with using the `EuiSuperSelect`. The reason `EuiSelect` seems a better choice is because `EuiSuperSelect` is usually used when we want to modify or customize the options inside the `select`. As this does not seem the case here, I think (maybe wrongly) that is better to have a simpler component with less complexity.

Reply:
yeah makes sense. We don't need any kind of customization here. We are simply showing text in dropdown. I will use `EuiSelect`.